cmd/agentbasedinstaller/import.go

```diff
@@ -65,7 +64,50 @@ func configureClusterIgnitionEndpoint(ctx context.Context, bmInventory *client.A
 	if updateClusterErr != nil {
 		return errorutil.GetAssistedError(updateClusterErr)
 	}
-	log.Infof("Configured ignition endpoint for cluster %s", *clusterID)
+	log.Infof("Updated configuration for imported cluster %s", *clusterID)
```

Comment:
`log.Infof("Configuration updated for imported cluster with ID: %s", *clusterID)`